apps/src/templates/sectionProgress/LoadMoreProgressButton.jsx

```diff
@@ -0,0 +1,28 @@
+import React, {Component} from 'react';
+import i18n from '@cdo/locale';
+import Button, {ButtonColor} from '@cdo/apps/templates/Button';
+
+const styles = {
+  container: {
+    clear: 'both',
+    padding: '10px 0px'
```

Comment:
Should padding be '10 0'?

Reply:
When it's in quotes, it's no longer a number and you need the px. Having it all on one line is a short cut, but then you do have to use string styles. This is equivalent to saying:
```
paddingTop: 10,
paddingBottom: 10,
paddingRight: 0,
paddingLeft: 0,
```

Reply:
Got it!